src/bayesnf/__init__.py

```diff
@@ -16,7 +16,7 @@
 
 # A new PyPI release will be pushed every time `__version__` is increased.
 # When changing this, also update the CHANGELOG.md
```

Comment:
do you want to keep the CHANGELOG? You could update that if so, or delete the file and this note if not!

Reply:
Removed now.